src/components/Tool/Home/Home.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useState } from 'react';
+import { makeStyles, Theme, Grid, Container, Box } from '@material-ui/core';
+import { motion } from 'framer-motion';
+
+import NavToolDrawer from '../NavToolList/NavToolDrawer';
+import Tool from './Tool';
+import { ToolsList } from './ToolList';
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
We kinda like the [styled api](https://material-ui.com/styles/basics/#styled-components-api) but this is ok too.